systems/render-pipeline/post-processing/final.js

```diff
@@ -0,0 +1,129 @@
+import { postProcessing as postprocessingShaders } from "pex-shaders";
+import { ssaoMixFlagDefinitions } from "./ssao.js";
+
+const finalFrag = postprocessingShaders.postProcessing.frag.replace(
```

Comment:
Any reason this replace is still in this PR? I thought we were pushing back changes to ssao til we're sure: https://github.com/pex-gl/pex-shaders/issues/14

Reply:
Rmoved